neuraxle/hyperparams/distributions.py

```diff
@@ -125,13 +130,13 @@ def __eq__(self, other):
 class FixedHyperparameter(HyperparameterDistribution):
     """This is an hyperparameter that won't change again, but that is still expressed as a distribution."""
 
-    def __init__(self, value):
+    def __init__(self, value, default_value=None):
```

Comment:
2 comments here: 

1. Some distributions return a `FixedHyperparameter` of themselves upon being `narrowed` (e.g.: choices). Please ensure that when `FixedHyperparameter` is called in those narrowings that the `default_value` is passed. 
2. You named it `default_value`. Change to `null_default_value` to be uniform.